libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -999,6 +1013,7 @@ void test_MQTT_Connect_happy_path()
     MQTT_GetIncomingPacketTypeAndLength_ReturnThruPtr_pIncomingPacket( &incomingPacket );
     MQTT_DeserializeAck_ExpectAnyArgsAndReturn( MQTTSuccess );
     MQTT_DeserializeAck_ReturnThruPtr_pSessionPresent( &sessionPresentExpected );
+    MQTT_PubrelToResend_ExpectAnyArgsAndReturn( MQTT_PACKET_TYPE_INVALID );
```

Comment:
Why was this needed to be added?

Reply:
Rebase brought the resend logic, so a call to function MQTT_PubrelToResend will be made if a session is present.